CONTRIBUTING.md

```diff
@@ -0,0 +1,133 @@
+   - **Purpose**: allows Developers and Data Engineers to easily choose and reuse individual samples depending on their requirements without the complexity of an end-to-end sample. Being small and modular also allows for showcasing different implementations for the same concept.
+
+3. [Utilities](/utilities/) - Reusable, standalone scripts that help with a specific operational task.
+   - **Purpose**: Easily find helpful scripts used in common tasks to manage a data engineering system.
+
+**Third**, decide which [key concepts](https://docs.microsoft.com/en-us/azure/architecture/framework/) are you showcasing in your sample. See [Microsoft Azure Well-Architected Framework](https://docs.microsoft.com/en-us/azure/architecture/framework/) for additional information. If you've decided to build a Single Technology Sample, decide which technology will be showcasing the key concepts.
+
+**Fourth**, locate where in the repository structure your sample will sit. Decide on the name of the sample -- short descriptive names are best. It is recommended to **not** choose a name too lengthy and complicated as it will be used across the repo (folder name, script paths, deep-linking in docs, labels, etc.).
+
+Repo Structure:
+
+```text
+/e2e_samples
+  /<name-of-e2e-sample-1>
+/single_tech_samples
+  /azuresql
+  /cosmosdb
+  /fabric
+  /databricks
+    /<name-of-singletech-sample-1>
+    /<name-of-singletech-sample-2>
+  ...
+/utilities
+  /fabric
+    /<name-of-utility-1>
+    /<name-of-utility-2>
+  /databricks
+```
+
+**Fifth**, Create a Github issue using the "New Sample Setup" template and fill in the information required. This issue template also contains a detailed checklist to guide you in your sample development.
+
+### Getting Started
+
+Once you have done all of the above, see this [sample structure](/docs/sample_project_structure/README.md) to get started. Use that structure as a starting point.
+
+#### Your sample's README
+
+This is arguably the *most important* part of your sample. In general, it is recommended to put most of the content in a single README to avoid content being "lost" and skipped over through deep linking.
+
+Structure of the README:
+
+1. **Table of Contents** - Use [this VSCode extension](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) to generate TOC.
+1. **Solution Overview** - Provide the background of the solution (i.e. What problem are you trying to solve?), solution architecture and key technologies used.
+1. **Key Learnings** - These are the key takeaways you want to impart to the engineer trying out the sample. For example, "Make your data pipelines replayable and idempotent".
+1. **Key Concepts** - These are the DataOps and WAF concepts that the sample is showcasing. For example, "Continuous Integration". This will be the meat of your sample.
```

Comment:
```suggestion
1. **Key Concepts** - These are the DataOps and WAF concepts that the sample is showcasing. For example, "Continuous Integration". This will form the core of your sample.
```